backend/app/donation/routes.py

```diff
@@ -114,3 +128,64 @@ def get_donation_amount():
 
         total_amount = sum([donation.amount for donation in filtered_donation])
         return jsonify(total_amount=total_amount)
+
+
+@donation.route("/csvs", methods=["POST"])
```

Comment:
how does `/donation-csv` sound? I have a feeling we might have to do something similar for contacts especially when we want to populate prod db and the fields would be different

Reply:
oh didn't know we were doing it for contacts as well, though this route is prefixed with `/donation` anyways as it is part of the `donation` blueprint. Should I still change it to `donation-csv`?

Reply:
oh right I totally forgot that's how Flask work

Reply:
csv is probs good? no strong opinions though csvs work too

Reply:
done